browser-extension/background.js

```
@@ -0,0 +1,150 @@
+}
+
+const api = new EmailServerAPI();
+
+// 监听来自content script和popup的消息
+chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+  (async () => {
+    switch (request.action) {
+      case 'login':
+        const loginResult = await api.login(request.username, request.password);
+        sendResponse(loginResult);
+        break;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)